ui/src/seminars/components/SeminarItem.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+
+import { VideoCameraAddOutlined } from '@ant-design/icons';
+import { Row, Col } from 'antd';
+import EditRecordAction from '../../common/components/EditRecordAction';
+import ResultItem from '../../common/components/ResultItem';
+import { SEMINARS } from '../../common/routes';
+import Title from '../../common/components/Title';
+import WebsitesAction from '../../common/components/WebsitesAction';
+import IconText from '../../common/components/IconText';
+import AuthorList from '../../common/components/AuthorList';
+import SeminarDatesAndTimes from './SeminarDatesAndTimes';
+import {
+  getLocalTimezone,
+  doTimezonesHaveDifferentTimes,
+} from '../../common/utils';
+
+function SeminarItem({ metadata, selectedTimezone }) {
+  const renderDatesInSelectedAndLocalTimezone = useCallback(
+    (startDate, endDate, localTimezone) => (
+      <>
+        <SeminarDatesAndTimes
+          startDate={startDate}
+          endDate={endDate}
+          timezone={selectedTimezone}
+          displayTimezoneAbbr
+          className="red"
+        />{' '}
+        (
+        <SeminarDatesAndTimes
+          startDate={startDate}
+          endDate={endDate}
+          timezone={localTimezone}
+          displayTimezoneAbbr
+        />
+        )
+      </>
+    ),
+    [selectedTimezone]
+  );
+  const title = metadata.get('title');
+  const recordId = metadata.get('control_number');
+  const canEdit = metadata.get('can_edit', false);
+  const urls = metadata.get('urls');
+  const joinUrls = metadata.get('join_urls');
+  const speakers = metadata.get('speakers');
+  const startDate = metadata.get('start_datetime');
+  const endDate = metadata.get('end_datetime');
+  const localTimezone = getLocalTimezone();
+  const timezoneDifferentThanLocal =
+    selectedTimezone &&
+    doTimezonesHaveDifferentTimes(selectedTimezone, localTimezone);
+  return (
+    <ResultItem
+      leftActions={
+        <>
+          {urls && <WebsitesAction websites={urls} />}
+          {joinUrls && (
+            <WebsitesAction
+              websites={joinUrls}
+              title={<IconText icon={<VideoCameraAddOutlined />} text="join" />}
+            />
+          )}
+          {canEdit && (
+            <EditRecordAction pidType="seminars" pidValue={recordId} />
+          )}
+        </>
+      }
+    >
+      <Row type="flex">
+        <Col>
+          <Link className="result-item-title" to={`${SEMINARS}/${recordId}`}>
+            <Title title={title} />
+          </Link>
+        </Col>
+      </Row>
+      <Row>
+        <Col>
+          <AuthorList authors={speakers} />
+        </Col>
+      </Row>
+      <Row>
+        <Col>
+          {timezoneDifferentThanLocal ? (
```

Comment:
actually, looks simpler like this.

```jsx
        { timezoneDifferentThanLocal  && (
        <SeminarDatesAndTimes
          startDate={startDate}
          endDate={endDate}
          timezone={selectedTimezone}
          displayTimezoneAbbr
          className="red"
        />{' '})
        <SeminarDatesAndTimes
          startDate={startDate}
          endDate={endDate}
          timezone={localTimezone}
          displayTimezoneAbbr={timezoneDifferentThanLocal}
        />
```

Reply:
I also need to put the second one in parenthesis if the condition is met. I could do another check and add the closing parenthesis but I think it looks more readable like this.

Reply:
ok that's fine, as long as the redundant function is gone.